configs/ote/custom-object-detection/cspdarknet_YOLOX/template.yaml

```
@@ -0,0 +1,55 @@
+# Description.
+model_template_id: Custom_Object_Detection_YOLOX
+name: YOLOX
+task_type: DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: Model with fastest inference speed, more than 2x compared to SSD, and comparable, slightly slower, training time.
```

```suggestion
summary: Model with fastest inference speed, compared to SSD, and comparable training time.
```
I'd avoid mentioning any performance numbers here, because those depend heavily on hardware which is not mentioned here, model precision and what not. IMO, that could potentially lead to biased expectations from the model